conf/config.sample.xml

```diff
@@ -178,6 +194,9 @@
             <js_module>defaultKwicConnect</js_module>
             <providers_conf extension-by="default">path/to/providers/conf.json</providers_conf>
         </kwic_connect>
+        <dispatch_hook>
```

Comment:
What's a `dispatch_hook`? should it be in this set of changes?

Reply:
`dispatch_hook` is the last plug-in we've created for you to be able to process Piwik logs. Soon, we at the CNC will be using the plug-in to improve our log processing too. This change is just a fix to keep the sample config up to date.